Makefile

```diff
@@ -69,7 +69,7 @@ endif
 # setup opencv
 ifeq ($(USE_OPENCV), 1)
 	CFLAGS += -DMXNET_USE_OPENCV=1 $(shell pkg-config --cflags opencv)
-	LDFLAGS += $(shell pkg-config --libs opencv)
+	LDFLAGS += $(filter-out -lopencv_ts, $(shell pkg-config --libs opencv))
```

Comment:
This keeps the gtest included in opencv build (when building with tests and performance tests) from giving link errors for a duplicate version of gtest